javascript-bank/account.js

```diff
@@ -0,0 +1,37 @@
+/* exported Account */
+
+var Account = function(number,holder) {
+  this.number = number;
+  this.holder = holder;
+  this.transactions = [];
+}
+
+Account.prototype.deposit = function (amount) {
+  if(amount > 0){
+    return true;
+  } else{
+    return false;
+  }
+}
+
+Account.prototype.withdraw = function (amount) {
+  if(amount > 0){
+    return true;
+  } else{
+    return false;
+  }
+}
+
+Account.prototype.getBalance = function () {
+  var total = 0;
+  console.log(this.transactions);
+  if(this.transactions.length === 0){
+    return 0;
+  }else{
+    this.transactions.forEach( v => {
+      total += v;
+    })
+  }
+  return total;
+}
```

Comment:
How is `getBalance` supposed to get the transaction totals if `deposit` and `withdraw` don't store transactions in the first place? Do the tests still pass? Please take and attach an updated screenshot whenever you finish addressing feedback to verify it still works:

---

![Screenshot_2](https://user-images.githubusercontent.com/16143771/111880227-612e6780-8967-11eb-88b2-331422af436e.jpg)